llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -90,16 +90,18 @@ static cl::opt<bool> OutputAssembly{"S",
 
 enum IRSplitMode {
   SPLIT_PER_TU,    // one module per translation unit
-  SPLIT_PER_KERNEL // one module per kernel
+  SPLIT_PER_KERNEL, // one module per kernel
+  SPLIT_AUTO // automatically select split mode
 };
 
 static cl::opt<IRSplitMode> SplitMode(
     "split", cl::desc("split input module"), cl::Optional,
     cl::init(SPLIT_PER_TU),
```

Comment:
```suggestion
    cl::init(SPLIT_AUTO),
```

Reply:
Changed in edfa44cc9a08c732d48f3bfe2b8caf3f8841d4ac